src/cts/test/array.ok

```diff
@@ -45,115 +45,26 @@ TritonCTS forced slew degradation on these wires.
     Sinks per sub-region: 140
     Sub-region size: 660.0000 X 350.4246
 [INFO CTS-0034]     Segment length (rounded): 176.
-    Key: 98 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 98 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 206 inSlew: 2 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 104 inSlew: 3 inCap: 1 outSlew: 1 load: 1 length: 8 delay: 9
-      location: 0.25 buffer: BUF_X4
-    Key: 98 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 98 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 206 inSlew: 2 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 104 inSlew: 3 inCap: 1 outSlew: 1 load: 1 length: 8 delay: 9
-      location: 0.25 buffer: BUF_X4
-    Key: 98 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 98 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 206 inSlew: 2 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 104 inSlew: 3 inCap: 1 outSlew: 1 load: 1 length: 8 delay: 9
-      location: 0.25 buffer: BUF_X4
-    Key: 98 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 98 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 206 inSlew: 2 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 104 inSlew: 3 inCap: 1 outSlew: 1 load: 1 length: 8 delay: 9
-      location: 0.25 buffer: BUF_X4
-    Key: 98 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 98 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 206 inSlew: 2 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 104 inSlew: 3 inCap: 1 outSlew: 1 load: 1 length: 8 delay: 9
-      location: 0.25 buffer: BUF_X4
-    Key: 98 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
-    Key: 98 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 8 delay: 1
  Level 2
```

Comment:
What about these messages? Can the user do something about these or are they also debug level?

Reply:
There isn't much user can do about these.   That's why these have been moved to debug messages.